CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 #### Breaking Changes
 
+- TraceMultiPropagator extension has been removed. Users should migrate to the instance returned 
+  from `TextMapPropagator.compositeWithShortCircuit()`. 
```

Comment:
were you going to remove this changelog entry?

Reply:
nm. looks like you haven't done anything with the comments yet. sorry for the noise!

Reply:
I mean, I said I was going to leave it for the release manager to wrangle, but if that's just noise I'm happy to remove it too.

Reply:
yeah, remove it for now, so it doesn't end up in the wrong place and I forget about it. :)